### PR TITLE
Add Content Hint API

### PR DESCRIPTION
**Pull request by enricop89**
Add Content Hint API for React Native. Documentation https://tokbox.com/developer/sdks/js/reference/OT.html#initPublisher

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,4 +19,11 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
-project.xcworkspace
\ No newline at end of file
+project.xcworkspace
```
Comment (abdulajet) [inline .gitignore:22]: Why do the project files need to be ignored? How does this affect development with a clean clone of the repo?

Comment (enricop89) [reply .gitignore:22]: I am not sure, let me check again

**Comment by enricop89** [comment]
@abdulajet I restored the gitignore file

**Review by abdulajet — APPROVED** [review]
lgtm

**Pull request merged**